index.html

```diff
@@ -0,0 +1,167 @@
+                </p>
+                <h3>Release date: October 2023</h3>
+                <p>Platform release: crunchyroll - wit studio NHK</p>
+            </div>
+        </section>
+    </header>
+    <main>
+        <section id="section1" class="section1">
+            <div class="cont-ti">
+                <h2>Main Program</h2>
+            </div>
+            <hr class="line">
+            <ul class="card-container">
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr1.png" alt="">
+                        <h3>Exhibition</h3>
+                        <p>Discover the "Latest Releases" in movies and series, find the most popular productions and receive recommendations.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr2.png" alt="">
+                        <h3>Forum</h3>
+                        <p>Participa en "Discusiones generales", analiza episodios, especula sobre tramas, lee reseñas de anime y comparte fanart y cosplay.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr3.png" alt="">
+                        <h3>Network</h3>
+                        <p> Create your profile, send private messages, join groups, check the events calendar and find friends with similar interests.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr4.png" alt="">
+                        <h3>Lecture</h3>
+                        <p>Browse the schedule, meet guest speakers, participate in Q&A sessions, and enjoy live streams.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr5.png" alt="">
+                        <h3>Workshop</h3>
+                        <p>Learn with featured instructors, sign up for specific times, and access additional resources and materials.</p>
+                    </div>
+                </li>
+            </ul>
+            <div class="cont-des">
+                <h2 class="mo"><a href="#">SEE MORE</a></h2>
+                <button type="button" class="des">Join into the world of anime</button>
+            </div>
+        </section>
+        <section class="section2">
+            <div class="title-section2">
+                <h2>Red carpet</h2>
+            </div>
+            <hr class="line">
+            <section>
+                <ul id="new-animes"></ul>
+            </section>
+            <div class="button-section2">
+                <button type="button">MORE<img src="ressources/icons/down.png"></button>
+            </div>
+        </section>
```

Comment:
- [ ] As stated in the project requirement The pages should look almost identical to the original design, outside the content differences. So Kindly I recommend you to  **adjust the anime section in three rows and two column**s just like the Figma design But in the case of your the anime section  are in two rows and four columns , So please adjust it.
- The Figma speaker section Look likes this : 
![speaker](https://github.com/AndresEspi/animePage/assets/93598189/3aa8e1e0-f723-42b3-88c0-690282544ec3)

- Your Anime section Look likes this  : 

![speaker](https://github.com/AndresEspi/animePage/assets/93598189/257cf516-e153-4376-8955-4cfb0ba0b0b4)